uptrain/operators/language/llm.py

```diff
@@ -50,7 +50,7 @@ async def async_process_payload(
         ):  # failed requests don't count towards rate limit
             try:
                 if payload.data["model"].startswith("gpt"):
-                    payload.response = await openai.ChatCompletion.acreate(
+                    payload.response = await openai.ChatCompletion.create(
```

Comment:
Hey Ashish, can you explain why you changed this function?

Reply:
Hey Dhruv, While doing development, was exploring some change here, missed to change it back.. Thanks for pointing it out.. Pushing another commit to revert this change.

Reply:
No problem. Can you do the same for lines 126, 135 and 137 in uptrain/operators/language/generation.py?

Reply:
Thanks for pointing it out. Pushed another commit for the same.